src/Controls/src/Core/MultiBinding.cs

```diff
@@ -76,6 +76,8 @@ internal override BindingBase Clone()
 			return clone;
 		}
 
+		internal static readonly object DoNothing = new object();
```

Comment:
Maybe we should add a comment that this moved in `Binding.cs`?

Reply:
I added a comment. I'm not sure if I did it the way you had in mind though 😄